maker/widget_maker.go

```diff
@@ -168,10 +168,10 @@ func MakeWidget(
 		widget = spotifyweb.NewWidget(app, pages, settings)
 	case "status":
 		settings := status.NewSettingsFromYAML(widgetName, moduleConfig, globalConfig)
-		widget = status.NewWidget(app, settings)
+		widget = status.NewWidget(app, pages, settings)
 	// case "system":
 	// 	settings := system.NewSettingsFromYAML(widgetName, moduleConfig, globalConfig)
-	// 	widget = system.NewWidget(app, date, version, settings)
```

Comment:
I've taken the settings widget out because I really didn't like how different the signature was, and the info is not very useful to end-users.